src/databricks/labs/ucx/contexts/workflow_task.py

```diff
@@ -81,6 +82,10 @@ def policies_crawler(self):
     def global_init_scripts_crawler(self):
         return GlobalInitScriptCrawler(self.workspace_client, self.sql_backend, self.inventory_database)
 
+    @cached_property
+    def py4j_table_crawler(self):
+        return FasterTableScanCrawler(self.sql_backend, self.inventory_database)
+
```

Comment:
Instead of this, can we update the `tables_crawler` property so that the correct crawler is selected automatically based on where the code is running?